### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by Zhang-bd** [qingma2016/DCTransformer#2]
Thank you for your response. Is the Harvard dataset processed using the same spectral response functions? If possible, could you provide a copy of the dataset?

**Comment by qingma2016** [comment]
The Harvard dataset uses the same spectral response function (SRF). The data I processed is too large to share conveniently, so I've provided a download link to the dataset in my paper. My preprocessing involved applying the SRF to the original HSIs to obtain the MSIs. For the training set, I applied Gaussian blurring to the original HSIs and stored the blurred HSIs in a separate folder (downsampling is performed during network training when loading the data). For the test set, I applied both Gaussian blurring and downsampling to the original HSIs.

**Comment by Zhang-bd** [comment]
Thank you for your patient answer. I noticed that you provided the link in your paper, but I found it unavailable. I would appreciate it if you could provide a link to the WV-2, GF5-S2B datasets on Baidu Web disk. Thank you again.

**Comment by qingma2016** [comment]
Here is the Baidu Cloud link to the WV-2 data set. Link: https://pan.baidu.com/s/1EhbNnsjt9yEvvq6XY1fzvg  Password: gig8
The GF5-S2B data set was shared with me by the collector of the data set. I am not sure whether he is willing to make it public, so I cannot share it with you.

**Comment by Zhang-bd** [comment]
Thank you for your answer. I will follow your work in the future. If there is any problem, I will contact you again.

**Comment by yzkqghxx** [comment]
How can MSI obtain it? I did not find the spectral response function file for Nikon D700 on the internet

**Comment by qingma2016** [comment]
Obtained using the "response_coefficient.mat" file.
